content/hi/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -0,0 +1,248 @@
+---
+
+title: macOS पर kubectl इंस्टॉल और सेट करें
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: macOS पर kubectl इंस्टॉल करें
+---
+
+## {{% heading "पूर्वाकांक्षित" %}}
+
+आपको kubectl संस्करण का उपयोग करना चाहिए जो आपके क्लस्टर के एक मामूली संस्करण अंतर के भीतर हो. उदाहरण के लिए,  v{{< skew latestVersion >}} क्लाइंट निम्नलिखित के साथ संवाद कर सकते हैं v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, and v{{< skew nextMinorVersion >}} कण्ट्रोल प्लेन.
+kubectl के नवीनतम संस्करण का उपयोग करने से अप्रत्याशित मुद्दों से बचने में मदद मिलती है.
+
+## macOS पर kubectl इंस्टॉल करें
+
+macOS पर kubectl संस्थापित करने के लिए निम्नलिखित विधियाँ मौजूद हैं :
+
+- [macOS पर curl के माध्यम से kubectl बाइनरी इंस्टॉल करें](#install-kubectl-binary-with-curl-on-macos)
+- [Homebrew का उपयोग करते हुए macOS पर इंस्टॉल करे](#install-with-homebrew-on-macos)
+- [Macports का उपयोग करते हुए macOS पर इंस्टॉल करे](#install-with-macports-on-macos)
```

Comment:
Hi. These links don't match the (translated) headings that they should link to.

Reply:
Hey @sftim, Those changes are suggested by @anubha-v-ardhan so I didn't change that. Should I change those links ?

Reply:
@anubha-v-ardhan I think I need to add header links there.

Reply:
Oops, Sorry
Yup You can change the links to translated headings
or else, you can also add these along headings in { }

Reply:
okay